spec/workers/step_by_step_scheduled_publish_worker_spec.rb

```diff
@@ -6,25 +6,10 @@
     allow(StepNavPublisher).to receive(:publish)
   end
 
-  let(:step_by_step_page) { create(:draft_step_by_step_page, scheduled_at: Time.now) }
-
-
-  it "publishes the step by step" do
-    described_class.new.perform(step_by_step_page.id)
-
-    expect(StepNavPublisher).to have_received(:publish).with(step_by_step_page)
-
-    step_by_step_page.reload
-    expect(step_by_step_page.has_been_published?).to be true
-  end
-
-  it "generates a change note" do
-    described_class.new.perform(step_by_step_page.id)
-    expect(step_by_step_page.internal_change_notes.first.description).to eq("Published on schedule")
-  end
+  let(:future_datetime) { Time.now + 1.hour }
```

Comment:
Can you move this change to a separate commit? Preferably when the "InFuture" validator was added? Or the commit directly after it.

Reply:
Done 👍